Samples/Forms/com.sentry.ovenlibrary/AndroidProject/OvenLibrary/src/androidTest/java/com/sentry/ovenlibrary/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.sentry.ovenlibrary;
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
```

Comment:
do you need these tests for a sample project? Seems like a dummy test created with the project